src/index.html

```diff
@@ -9,10 +9,50 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="iMac"
```

Comment:
alt attribute should describe the image content
<img width="471" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/63243136/8377b064-28b7-4c02-945a-fe5eb68aa79e">